README.md

```diff
@@ -1,254 +1,422 @@
-OK
+## <a name="usage"> Using the Conjur Service Broker
 
-service             plans                  description
-cyberark-conjur     community              An open source security service that provides secrets management, machine-identity based authorization, and more.
+- [Creating a Service Instance](#prepare-space)
+- [Connecting an Application to Conjur](#connect-app)
+- [Rotating App Host Credentials](#rotate-credentials)
 
-TIP:  Use 'cf marketplace -s SERVICE' to view descriptions of individual plans of a given service.
-```
+### <a name="prepare-space"> Creating a Conjur Service Instance
 
-When you are ready to use the Conjur Service Broker, you can create a Conjur
-service instance under the `community` plan in the org / space where you will be
-deploying your application:
-```
-cf create-service cyberark-conjur community conjur
-```
+1. **Confirm Conjur Service is in the Marketplace**
+
+    Once the Service Broker is installed, you should see the service listing from any
+    org / space:
+
+    ```sh-session
+    $ cf marketplace
+    Getting services from marketplace in org cyberark-conjur-org / space cyberark-conjur-space as admin...
+    OK
+
+    service             plans                  description
+    cyberark-conjur     community              An open source security service that provides secrets management, machine-identity based authorization, and more.
+
+    TIP:  Use 'cf marketplace -s SERVICE' to view descriptions of individual plans of a given service.
+    ```
 
-Note that service instances can not be shared between spaces, so a Conjur
-service instance must be created in each space with apps that retrieve secrets
-from Conjur.
-
-For PCF 2.0+, when the service broker is provisioned in a space, it will automatically create
-a policy branch for the org and space if it doesn't already exist. The policy will look similar to
-this:
-
-```yaml
----
-# Policy for the Organization
-- !policy
-  # Organization GUID from PCF.
-  # This may be obtained by running `cf org --guid {org name}
-  id: cbd7a05a-b304-42a9-8f66-6827ae6f78a1
-  body:
-    # Layer to allow privileging an entire organzation to a resource
-    - !layer
-
-    # Policy for the Space
+2. **Create a Conjur Service Instance**
+
+    When you are ready to use the Conjur Service Broker, you can create a Conjur
+    service instance under the `community` plan in the org / space where you are
+    deploying your application:
+    ```
+    cf create-service cyberark-conjur community conjur
+    ```
+
+    > **NOTE:** Service instances cannot be shared between spaces. A Conjur
+    service instance must be created in each space where apps retrieve secrets
+    from Conjur.
+
+    For PCF 2.0+, when the service broker is provisioned in a space, it automatically creates
+    a policy branch for the org and space if it doesn't already exist. The policy looks similar to
+    this:
+
```

Comment:
We probably need something here that says if the flag is on, a space-level host will also be created on provision.

Reply:
Maybe we can refer to the section below on the app identity.

Reply:
Actually, I'm in favor of moving the "app host vs space host" and "entitling your app" sections up here.